containers/compose.yaml

```diff
@@ -32,7 +32,6 @@ services:
       interval: 10s
       timeout: 5s
       retries: 10
-    command: -p ${PSQL_PORT}
```

Comment:
@esloch found a duplicity here, is it ok to remove? It was breaking the docker-compose command